ee/watchdog/controller_windows.go

```diff
@@ -153,6 +158,17 @@ func (wc *WatchdogController) Interrupt(_ error) {
 
 func (wc *WatchdogController) ServiceEnabledChanged(enabled bool) {
 	ctx := context.TODO()
+	// we don't alter watchdog installation (install or remove) if this is a non-prod deployment
```

Comment:
We could probably also check if we're running as not-root.

Reply:
oh good call, on it!